shared/desktop/yarn-helper/font.js

```diff
@@ -66,16 +66,7 @@ const getSvgPaths = shouldPrintSkipped =>
  *
  * For config options: https://github.com/sunflowerdeath/webfonts-generator
  */
-function updatedFonts() {
-  let webfontsGenerator
-  try {
-    webfontsGenerator = require('webfonts-generator')
```

Comment:
this require needs to exist still for this to work. i moved it down here so its not required for people who aren't doing font stuff (it moved to an optional dep)

Reply:
Thanks for the catch here. I think I over zealously removed some code - did not do this one intentionally.